_projects/view/accelerating-your-ultra96/README.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Accelerating Your Ultra96 Developments
+permalink: "/projects/accelerating-your-utra96/"
```

Comment:
I think there is a typo here - it should be:

`/projects/accelerating-your-ultra96/`